client/src/app/site/services/duration.service.ts

```diff
@@ -79,7 +79,7 @@ export class DurationService {
         const minutes = `0${Math.floor((duration % 3600) / 60)}`.slice(-2);
         const seconds = `0${Math.floor(duration % 60)}`.slice(-2);
         if (!isNaN(+minutes) && !isNaN(+seconds)) {
```

Comment:
Seconds is not needed anymore. 

```suggestion
        const minutes = `0${Math.floor((duration % 3600) / 60)}`.slice(-2);
        if (!isNaN(+minutes)) {
```

Reply:
done.